ush/wave_prnc_ice.sh

```diff
@@ -55,7 +55,7 @@ source "$HOMEgfs/ush/preamble.sh"
   echo "Making ice fields."
 
   if [[ -z "${YMDH}" ]] || [[ -z "${cycle}" ]] || \
-     [[ -z "${COM_WAVE_PREP}" ]] || [[ -z "${FIXwave}" ]] || [[ -z "${EXECwave}" ]] || \
+     [[ -z "${COM_WAVE_PREP}" ]] || [[ -z "${FIXgfs}" ]] || [[ -z "${EXECwave}" ]] || \
```

Comment:
should this be `${FIXgfs}/wave`?

Reply:
So it was `${FIXgfs}/wave` but I had this shellcheck warning: https://github.com/NOAA-EMC/global-workflow/security/code-scanning/11613. I removed the `/wave` part to resolve the warning. I think the if-block in question can actually be removed, it seems like it's unnecessary. I will have to touch this block again when I go to remove/replace things like `EXECwave` so I could look at removing those `-z` checks then. Thoughts on that block?